core/src/main/resources/error/error-classes.json

```diff
@@ -248,11 +248,21 @@
           "The <exprName> must not be null"
         ]
       },
+      "UNEXPECTED_NUM_PARAMS" : {
```

Comment:
It seems it is similar to `WRONG_NUM_PARAMS`. Could you re-use the existing one or improve it.

Reply:
I think we can make `WRONG_NUM_PARAMS` more general

Reply:
Can I change it to：
"WRONG_NUM_PARAMS" : {
        "message" : [
          "The <functionName> requires <num> arguments (actual number = <actualNum>)"
        ]
      }

Reply:
Please, open a separate PR and change the error template. I would change it to:
```json
      "WRONG_NUM_PARAMS" : {
        "message" : [
          "The <functionName> requires <expectedNum> parameters but the actual number is <actualNum>."
        ]
      }
```

Reply:
ok